packages/database/lib/schema.ts

```diff
@@ -605,19 +606,20 @@ export const reactions = pgTable(
 export const users = pgTable(
   "users",
   {
-    // Just the pub key of anything that interacts with the system.
-    // Eventually want to add this constraint to the other tables, but for now
-    // want to see how it feels.
-    userAcct: pubkey("user_acct").notNull(),
-    createdAt: timestamp("created_at")
-      .notNull()
-      .default(sql`now()`),
-  },
-  (table) => ({
-    userUnique: unique("unique_user").on(table.userAcct),
-  })
+    userAcct: pubkey("user_acct").primaryKey(),
+    createdAt: timestamp("created_at").notNull().default(sql`now()`),
+  }
 );
 
+export const sessions = pgTable(
+  "sessions",
+  {
+    id: uuid("id").primaryKey(),
+    userAcct: pubkey("user_acct").references(() => users.userAcct, {onDelete: "restrict", onUpdate: "restrict"}),
```

Comment:
add another column for createdAt (created_at) just as a frame of reference and that can be NOT null